Commands/CalcActivateSheetCommand.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using G1ANT.Language;
+
+namespace G1ANT.Addon.LibreOffice.Commands
+{
+    [Command(Name = "calc.activatesheet", Tooltip = "Activate the specified sheet")]
+    class CalcActivateSheetCommand : Command
+    {
+       public CalcActivateSheetCommand(AbstractScripter scripter) : base(scripter)
+        {
+
+        }
```

Comment:
If constructor is empty move the curly braces to one line => `{ }` here and elsewhere.